net/pdnsd/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2019 Xiaobo Tian
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pdnsd
+PKG_VERSION:=1.2.9a
+PKG_RELEASE=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-par.tar.gz
+PKG_SOURCE_URL:=http://members.home.nl/p.a.rombouts/$(PKG_NAME)/releases/
+PKG_HASH:=bb5835d0caa8c4b31679d6fd6a1a090b71bdf70950db3b1d0cea9cf9cb7e2a7b
+
+PKG_MAINTAINER:=Xiaobo Tian <peterwillcn@gmail.com>
+PKG_LICENSE:=GPL-2.0+
```

Comment:
Change to GPL-2.0-or-later, as per SPDX: https://spdx.org/licenses/

Reply:
Actually the license file says GPL-3.0-or-later

Reply:
Also add PKG_LICENSE_FILES:=COPYING

Reply:
Fixed